docs/playlist.md

```diff
@@ -173,8 +173,30 @@ kalturaPlayer.playlist.playItem(lastItemIndex);
 
 For the complete `playlist` API, see [PlaylistManager](./api.md#playlistmanager).
 
+## Playlist Events
+
+Once a playlist load is requested async calls will be called until the playlist object is ready for use. Inorder to use the api correctly you must be aware that the playlist is ready for use after the playtist is loaded (witing for `PLAYLIST_LOADED` event).
+
+[Playlist Events](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/playlist/playlist-event-type.js)
+
+
+ * `PLAYLIST_LOADED`
+ * `PLAYLIST_ITEM_CHANGED` (multiple)
+ * `PLAYLIST_ENDED`
+
+
+## Listening to Event before calling playlist API the first time
```

Comment:
```suggestion
## Listening to an Event before calling the playlist API the first time
```

Reply:
"for the first time" instead of "at the first time"